src/pybind/mgr/selftest/module.py

```diff
@@ -311,10 +311,11 @@ def _test_remote_calls(self):
         self.remote("influx", "handle_command", "", {"prefix": "influx self-test"})
 
         # Test calling module that exists but isn't enabled
+        # (arbitrarily pick a non-always-on module to use)
+        disabled_module = "telegraf"
         mgr_map = self.get("mgr_map")
-        all_modules = [m['name'] for m in mgr_map['available_modules']]
-        disabled_modules = set(all_modules) - set(mgr_map['modules'])
-        disabled_module = list(disabled_modules)[0]
+        assert disabled_moduled not in mgr_map['modules']
```

Comment:
`disabled_module`d typo?

Reply:
also: presumably the the assert is testing that the module isn't enabled and it isn't always on. might want to test that its not in the union of map['modules'] and map['always_on_modules']